CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ This file also provides links to Jenkins versions,
 which bundle the specified remoting version.
 See [Jenkins changelog](https://jenkins.io/changelog/) for more details.
 
+##### 3.16
+
+Release date: Jan 10, 2018 => Jenkins 2.102
+
+* [PR #190](https://github.com/jenkinsci/remoting/pull/190) -
```

Comment:
Is it really 190 ? since it seems to be about certificate check and no trace of ClassFilter inside the PR. It seems to be more 208 no ?